DESCRIPTION

```diff
@@ -1,57 +1,62 @@
-Video: https://www.youtube.com/watch?v=2KqBSbkfhRo , https://www.youtube.com/watch?v=3PK_jx44QTs
-BugReports: https://github.com/Bioconductor/GenomicAlignments/issues
-Version: 1.37.0
-License: Artistic-2.0
-Encoding: UTF-8
-Authors@R: c(
-	person("Hervé", "Pagès", role=c("aut", "cre"),
-		email="hpages.on.github@gmail.com"),
-	person("Valerie", "Obenchain", role="aut"),
-	person("Martin", "Morgan", role="aut"),
-	person("Robert", "Castelo", role="ctb"))
-Depends: R (>= 4.0.0), methods, BiocGenerics (>= 0.37.0),
-	S4Vectors (>= 0.27.12), IRanges (>= 2.23.9), GenomeInfoDb (>= 1.13.1),
-	GenomicRanges (>= 1.41.5), SummarizedExperiment (>= 1.9.13),
-	Biostrings (>= 2.55.7), Rsamtools (>= 1.31.2)
-Imports: methods, utils, stats, BiocGenerics, S4Vectors,
-	IRanges, GenomicRanges, Biostrings, Rsamtools, BiocParallel
-LinkingTo: S4Vectors, IRanges
-Suggests: ShortRead, rtracklayer, BSgenome, GenomicFeatures,
-	RNAseqData.HNRNPC.bam.chr14, pasillaBamSubset,
-	TxDb.Hsapiens.UCSC.hg19.knownGene, TxDb.Dmelanogaster.UCSC.dm3.ensGene,
-	BSgenome.Dmelanogaster.UCSC.dm3, BSgenome.Hsapiens.UCSC.hg19,
-	DESeq2, edgeR,
-	RUnit, BiocStyle
-Collate: utils.R
-	cigar-utils.R
-	GAlignments-class.R
-	GAlignmentPairs-class.R
-	GAlignmentsList-class.R
-	GappedReads-class.R
-	OverlapEncodings-class.R
-	findMateAlignment.R
-	readGAlignments.R
-	junctions-methods.R
-	sequenceLayer.R
-	pileLettersAt.R
-	stackStringsFromGAlignments.R
-	intra-range-methods.R
-	coverage-methods.R
-	setops-methods.R
-	findOverlaps-methods.R
-	coordinate-mapping-methods.R
-	encodeOverlaps-methods.R
-	findCompatibleOverlaps-methods.R
-	summarizeOverlaps-methods.R
-	findSpliceOverlaps-methods.R
-	zzz.R
+Package: GenomicAlignments
+Title: Representation and manipulation of short genomic alignments
+Description: Provides efficient containers for storing and manipulating
+	short genomic alignments (typically obtained by aligning short reads
+	to a reference genome). This includes read counting, computing the
+	coverage, junction detection, and working with the nucleotide content
+	of the alignments.
+biocViews: Infrastructure, DataImport, Genetics, Sequencing, RNASeq, SNP,
+	Coverage, Alignment, ImmunoOncology
+URL: https://bioconductor.org/packages/GenomicAlignments
+Video: https://www.youtube.com/watch?v=2KqBSbkfhRo , https://www.youtube.com/watch?v=3PK_jx44QTs
+BugReports: https://github.com/Bioconductor/GenomicAlignments/issues
+Version: 1.37.0
+License: Artistic-2.0
+Encoding: UTF-8
+Authors@R: c(
+	person("Hervé", "Pagès", role=c("aut", "cre"),
+		email="hpages.on.github@gmail.com"),
+	person("Valerie", "Obenchain", role="aut"),
+	person("Martin", "Morgan", role="aut"),
+	person("Robert", "Castelo", role="ctb"),
+	person("Rohit", "Satyam", role = "ctb",
+        comment = "Converted summarizeOverlaps vignette from Sweave to RMarkdown / HTML."),
+  person("Bioconductor Package Maintainer",
+        email = "maintainer@bioconductor.org", role = "cre"))

```

Comment:
Herve is the maintainer, so remove lines 24-25. 

Line 23 should be  formatted like `email` on line 18. It should have the same indent and no spaces around the `=`.

Replace the ending comma with a closing parenthesis.